week3/helpMenu.txt

```diff
@@ -0,0 +1,63 @@
+
+    HELP MENU                                                                                    ||
+=================                                                                                ||
+                                                                                                 ||
+Instructions:                                                                                    ||
+                                                                                                 ||
+---------------------------------------                                                          ||
+$ curl -X GET 'http://localhost:8080/' >>>>>>>>>>> To display the help menu                      ||
+---------------------------------------                                                          ||
+                                                                                                 ||
+---------------------------------------                                                          ||
+$ curl -X GET 'http://localhost:8080/help' >>>>>>>>>>> To display the help menu                  ||
+---------------------------------------                                                          ||
+                                                                                                 ||
+-----------------------------------------------------                                            ||
+$ curl -X POST 'http://localhost:8080/add/{newTask}' >>>>>>>>>>> To add a task to the list,      ||
```

Comment:
This is interesting, but not a normal way to pass data in a POST request. The URL path is used to specify location - for example, if we had `todos` but also `calendarEvents`, we could use the path to differentiate.
You should pass the data of the task to add in the "body" of the post request (and then use `bodyParser` to get it out of the body inside your `app.post()` handler.